nodebuilder/state/cmd/state.go

```diff
@@ -156,47 +153,6 @@ var submitTxCmd = &cobra.Command{
 	},
 }
 
-var submitPFBCmd = &cobra.Command{
```

Comment:
is this meant to be part of this PR or it just contains commits from the deprecating submitPFB?

Reply:
I think we can leave it here. Removing all at once

Reply:
I would ask @jcstein if this is okay first

Reply:
The target branch is a feature branch, so I guess it's fine. But sure, let's wait @jcstein

Reply:
if I understand correctly and this PR is to deprecate submitPFB as a gateway endpoint, we should make sure this is tracked properly with other breaking RPC changes and let teams know accordingly. but I do think this is okay to deprecate

that being said, while i will be updating docs, i cannot speak for everyone using the submitPFB endpoint still. so i will cc @Ferret-san and @YazzyYaz in case there are any teams that you know of still using this. we will have to let them know that this is deprecated when it is in a release

it is also important that the release is made properly as a breaking release, and not just another RC that breaks everything that worked in the previous RC on same semantic version i.e. v0.11.0-rc1 to v0.11.0-rc2 -- unless I am misunderstanding how breaking changes should be handled between RCs 🫡

Reply:
Please note, that this PR will be merged into a feature branch and not main.

Reply:
>it is also important that the release is made properly as a breaking release, and not just another RC that breaks everything that worked in the previous RC on same semantic version i.e. v0.11.0-rc1 to v0.11.0-rc2 -- unless I am misunderstanding how breaking changes should be handled between RCs 🫡

@jcstein we've been working towards a stable release which will be v0.11.0 and expected in a few weeks. Every RC up to that point has been / will likely be breaking in some way or another but we will 1. absolutely document when it is, and 2. we are making feature branches for breaks so that we get **all** the known API breaks that we want to do all in one go (one RC)

Reply:
sounds good! thank you for clarifying @vgonkivs and @renaynay. i am realizing that since we put these callouts in months ago to warn people that gateway will be deprecated, it should be no surprise and we will communicate accordingly
<img width="1002" alt="scrot 2023-09-28 at 9 43 04 AM" src="https://github.com/celestiaorg/celestia-node/assets/46639943/37368b52-5163-4bbb-89aa-9dcaeddcc49a">

Reply:
i've made an issue to track this as "unreleased node" https://github.com/celestiaorg/docs/issues/1094